src/mom6-tools.plot/README.md

```diff
@@ -0,0 +1,31 @@
+0-1. This directory contains python plotting scripts based on mom6-tools. Note that scripts run with local cartopy shapefiles in /scratch2/NCEPDEV/marineda/common/cartopy.
+
+0-2. Python scripts require two input arguments: grid and data file names. Multiple data files can be used with wildcard (*) input for data file names. As of now (01/07/2020), diag_table files are not fully updated for variables needed in mom6-tools modules yet. For the time being, users can utilize ocean_geometry.nc in /scratch2/NCEPDEV/marineda/common. Test runs for plotting examples are:
+
+python ice.plot.py -grid ocean_geometry.nc -data cic.socagodas.an.2011-10-*T12:00:00Z.nc -figs_path ./fcst -var hice aice
+
+-grid file: geometry input file should contain geolon and geolat variables.
+
+-data file: aice and hice variables are used to make hemispheric plots.
+
+-figs_path (optional): path to save png files (e.g., -figs_path ./fcst)
+
+-var (optional): variable names to plot (e.g., -var hice aice or -var hi_h aice_h)
```

Comment:
Are there any variables as hi_h and aice_h?
These vaiables were not found in cic.socagodas.an.2011-10-01T12:00:00Z.nc while testing.

Is this a typo?